goal_src/jak1/pc/pckernel-h.gc

```diff
@@ -426,7 +470,8 @@
   (set! (-> obj vsync?) #t)
   (set! (-> obj letterbox?) #t)
 
-  (pc-get-screen-size -1 (&-> obj width) (&-> obj height))
+  ;; TODO - why do we change the windowed mode twice
```

Comment:
`set-size!` performs a different action depending on whether we are in windowed mode or not. It's much simpler than having 2 separate functions everywhere we want to set game resolution.